app/controllers/application_controller.rb

```diff
@@ -106,6 +106,10 @@ def can_administer?
     current_user&.admin?
   end
 
+  def count_and_pluralize(results)
```

Comment:
Pluralization should be handled by the I18n system rather than a custom method. See, for example, the `requested_accounts_alert` messages in `en.yml` and how they are used.